salt/utils/__init__.py

```diff
@@ -2283,3 +1613,174 @@ def dns_check(addr, port, safe=False, ipv6=None):
         'Salt Oxygen. This warning will be removed in Salt Neon.'
     )
     return salt.utils.network.dns_check(addr, port, safe, ipv6)
+
+
+def get_context(template, line, num_lines=5, marker=None):
+    # Late import to avoid circular import.
+    import salt.utils.versions
+    import salt.utils.templates
+    salt.utils.versions.warn_until(
+        'Neon',
+        'Use of \'salt.utils.get_context\' detected. This function '
+        'has been moved to \'salt.utils.templates.get_context\' as of '
+        'Salt Oxygen. This warning will be removed in Salt Neon.'
+    )
+    return salt.utils.templates.get_context(template, line, num_lines, marker)
+
+
+def get_master_key(key_user, opts, skip_perm_errors=False):
+    # Late import to avoid circular import.
+    import salt.utils.versions
+    import salt.utils.master
+    salt.utils.versions.warn_until(
+        'Neon',
+        'Use of \'salt.utils.get_master_key\' detected. This function '
+        'has been moved to \'salt.utils.master.get_master_key\' as of '
+        'Salt Oxygen. This warning will be removed in Salt Neon.'
+    )
+    return salt.utils.master.get_master_key(key_user, opts, skip_perm_errors)
+
+
+def get_values_of_matching_keys(pattern_dict, user_name):
+    # Late import to avoid circular import.
+    import salt.utils.versions
+    import salt.utils.master
+    salt.utils.versions.warn_until(
+        'Neon',
+        'Use of \'salt.utils.get_values_of_matching_keys\' detected. '
+        'This function has been moved to \'salt.utils.master.get_master_key\' '
```

Comment:
`get_master_key` doesn't match the call to `salt.utils.master.get_values_of_matching_keys` below.